openam-federation/openam-federation-library/src/main/java/com/sun/identity/saml2/common/SAML2Constants.java

```diff
@@ -809,12 +809,12 @@ public interface SAML2Constants {
     public String CLASS_REF_AUTH_TYPE = "AuthContextClassRef";
 
     public String AUTH_CONTEXT_DECL_REF ="AuthContextDeclRef";
-    public String AUTH_CONTEXT_DECL_REF_ATTR 
+    public String AUTH_CONTEXT_DECL_REF_ATTR
                                         ="sunFMAuthContextDeclareRef";
 
     public String AUTH_CONTEXT_CLASS_REF ="AuthnContextClassRef";
```

Comment:
It is very much out-of-scope for this PR, but with all the whitespace changes I would like to fix formatting errors as well.

Reply:
I have reverted the changes in this file, as I did not change anything (only formatting). Reformatting of files should be done in a separate issue.